emily_cron/app/models/deposit.py

```diff
@@ -66,7 +66,7 @@ def max_fee(self) -> int:
     @functools.cached_property
     def deposit_time(self) -> int:
```

Comment:
maybe nit: `deposit_time` is a bit misleading as it could suggest (given the usage below) it's the creation time, maybe `deposit_last_update` or similar could be a more precise name